roles/rclone/tasks/main.yml

```diff
@@ -0,0 +1,30 @@
+---
+- name: Uninstall current version of rclone
+  become: yes
+  shell: rm /usr/bin/rclone /usr/local/share/man/man1/rclone.1 '{{ rclone_config_location }}/rclone.conf'
```

Comment:
If you swap to the repository versions (previous comment) this will be unnecessary